ui/de-lib/src/main/java/org/iplantc/de/client/util/OntologyUtil.java

```diff
@@ -69,7 +71,16 @@ public Avu convertHierarchyToAvu(OntologyHierarchy hierarchy) {
         Avu avu = avuFactory.getAvu().as();
         avu.setAttr(getAttr(hierarchy));
         avu.setValue(hierarchy.getIri());
-        avu.setUnit(hierarchy.getLabel());
+        avu.setUnit("");
```

Comment:
It looked to me like Unit wasn't optional, so I'm passing an empty string here. Let me know if this needs to change.